Tests/Test scenes.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I wrote about the Test scene [here](https://github.com/Unity-Technologies/MeshSyncDCCPlugins/pull/288#issuecomment-1292133617) before.

Copy pasting it here.

Files under the `Tests` folder are supposed to be used for CI package tests.
For test assets, I would recommend to put them in another folder, such as `TestAssets` or `Plugins~/TestAssets`

Reply:
Done